### PR TITLE
[BEAM-1741] Update direct and Cloud Dataflow runner pages with Python info

### PR DESCRIPTION
**Pull request by melap**
R: @aaltay

### CODE REVIEW

File: src/documentation/runners/dataflow.md [hunk 1]
```diff
@@ -51,6 +58,8 @@ You must specify your dependency on the Cloud Dataflow Runner.
 </dependency>
 ```
 
+<span class="language-python">When using Python, you do not need to specify your dependency on the Cloud Dataflow Runner.</span>
```
Comment (aaltay) [inline src/documentation/runners/dataflow.md]: Do we need this? It is a sentence about doing nothing.

File: src/documentation/runners/dataflow.md [hunk 2]
```diff
@@ -61,44 +70,83 @@ gcloud auth application-default login
...
+<tr class="language-python">
+  <td><code>save_main_session</code></td>
+  <td>Save the main session state so that pickled functions and classes defined in <code>__main__</code> (e.g. interactive session) can be unpickled. Some workflows do not need the session state if, for instance, all of their functions/classes are defined in proper modules (not <code>__main__</code>) and the modules are importable in the worker.</td>
+  <td><code>false</code></td>
+</tr>
+
+<!-- Only show for Python -->
+<tr class="language-python">
+  <td><code>sdk_location</code></td>
+  <td>Override the default GitHub location from where the Cloud Dataflow SDK is downloaded. This value can be an URL, a Cloud Storage path, or a local path to an SDK tarball. Workflow submissions will download or copy the SDK tarball from this location. If set to the string <code>default</code>, a standard SDK location is used. If empty, no SDK is copied.</td>
```
Comment (aaltay) [inline src/documentation/runners/dataflow.md]: This description needs to be fixed in pipeline options as well. Could you change this to:
```
Override the default location from where the Beam SDK is downloaded. This value can be an URL, a Cloud Storage path, or a local path to an SDK tarball. Workflow submissions will download or copy the SDK tarball from this location. If set to the string <code>default</code>, a standard SDK location is used. If empty, no SDK is copied.
```

File: src/documentation/runners/dataflow.md [hunk 3]
```diff
@@ -111,8 +159,11 @@ While your pipeline executes, you can monitor the job's progress, view details o
...
+To connect to your job and block until it is completed, call <span class="language-java"><code>waitToFinish</code></span><span class="language-python"><code>wait_until_finish</code></span> on the `PipelineResult` returned from `pipeline.run()`. The Cloud Dataflow Runner prints job status updates and console messages while it waits. While the result is connected to the active job, note that pressing **Ctrl+C** from the command line does not cancel your job. To cancel the job, you can use the [Dataflow Monitoring Interface](https://cloud.google.com/dataflow/pipelines/dataflow-monitoring-intf) or the [Dataflow Command-line Interface](https://cloud.google.com/dataflow/pipelines/dataflow-command-line-intf).
```
Comment (aaltay) [inline src/documentation/runners/dataflow.md]: What is the meaning of `to connect` here?

File: src/documentation/runners/direct.md [hunk 4]
```diff
@@ -33,9 +47,11 @@ You must specify your dependency on the Direct Runner.
...
 ```
 
+<span class="language-python">When using Python, you do not need to specify your dependency on the Direct Runner.</span>
+
 ## Pipeline options for the Direct Runner
 
-When executing your pipeline from the command-line, set `runner` to `direct`. The default values for the other pipeline options are generally sufficient.
+When executing your pipeline from the command-line, set `runner` to `direct` or `DirectRunner`. The default values for the other pipeline options are generally sufficient.
 
 See the reference documentation for the  <span class="language-java">[`DirectOptions`]({{ site.baseurl }}/documentation/sdks/javadoc/{{ site.release_latest }}/index.html?org/apache/beam/runners/direct/DirectOptions.html)</span><span class="language-python">[`PipelineOptions`](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/utils/pipeline_options.py)</span> interface (and its subinterfaces) for defaults and the complete list of pipeline configuration options.
```
Comment (aaltay) [inline src/documentation/runners/direct.md]: There is also `DirectOptions`, that is probably the better link here. https://beam.apache.org/documentation/sdks/pydoc/0.6.0/apache_beam.utils.html#apache_beam.utils.pipeline_options.DirectOptions

(Assuming that we are talking about direct runner specific options.)

File: src/documentation/runners/dataflow.md (hunk 1 above)
Comment (melap) [inline src/documentation/runners/dataflow.md]: changed to a more generic "not applicable" sentence

File: src/documentation/runners/dataflow.md (hunk 2 above)
Comment (melap) [inline src/documentation/runners/dataflow.md]: done

File: src/documentation/runners/dataflow.md (hunk 3 above)
Comment (melap) [inline src/documentation/runners/dataflow.md]: changed to "To block until your job completes, ..."

File: src/documentation/runners/direct.md (hunk 4 above)
Comment (melap) [inline src/documentation/runners/direct.md]: done

**Comment by melap** [comment]
Thanks for your feedback! I made some changes, PTAL

File: src/documentation/runners/dataflow.md [hunk 5]
```diff
@@ -61,47 +70,89 @@ gcloud auth application-default login
...
+<tr class="language-python">
+  <td><code>save_main_session</code></td>
+  <td>Save the main session state so that pickled functions and classes defined in <code>__main__</code> (e.g. interactive session) can be unpickled. Some workflows do not need the session state if, for instance, all of their functions/classes are defined in proper modules (not <code>__main__</code>) and the modules are importable in the worker.</td>
+  <td><code>false</code></td>
+</tr>
+
+<!-- Only show for Python -->
+<tr class="language-python">
+  <td><code>sdk_location</code></td>
+  <td>Override the default location from where the Cloud Dataflow SDK is downloaded. This value can be an URL, a Cloud Storage path, or a local path to an SDK tarball. Workflow submissions will download or copy the SDK tarball from this location. If set to the string <code>default</code>, a standard SDK location is used. If empty, no SDK is copied.</td>
```
Comment (aaltay) [inline src/documentation/runners/dataflow.md]: please use Beam SDK instead of Cloud Dataflow SDK

Comment (melap) [inline src/documentation/runners/dataflow.md]: ACK! good catch, not sure how I managed to revert that. FIXED!

**Comment by aaltay** [comment]
LGTM

**Comment by melap** [comment]
SMall update to this en route, been doing battle with the javascript switcher and need to update all runner pages to specifically use language-py.

**Pull request closed without merging**